<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostedCreatePostDialog</class>
 <widget class="QDialog" name="PostedCreatePostDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Submit Post</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="HeaderFrame" name="headerFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="3">
       <widget class="StyledLabel" name="info_label">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>178</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>178</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>178</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>178</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>You are submitting a link. The key to a successful submission is interesting content and a descriptive title.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QLineEdit" name="titleEdit"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="titleLabel">
          <property name="text">
           <string>Title</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="linkLabel">
          <property name="text">
           <string>Link</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="linkEdit"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="spacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>78</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Signed by: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="GxsIdChooser" name="idChooser"/>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_NewId">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>Create new Identity</string>
          </property>
          <property name="text">
           <string>New ID</string>
          </property>
          <property name="icon">
           <iconset resource="../images.qrc">
            <normaloff>:/images/identity/identity_create_32.png</normaloff>:/images/identity/identity_create_32.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0" colspan="3">
       <layout class="QVBoxLayout" name="notesLayout">
        <item>
         <widget class="QLabel" name="notesLabel">
          <property name="text">
           <string>Notes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="notesTextEdit"/>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="submitButton">
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HeaderFrame</class>
   <extends>QFrame</extends>
   <header>gui/common/HeaderFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GxsIdChooser</class>
   <extends>QComboBox</extends>
   <header>gui/gxs/GxsIdChooser.h</header>
  </customwidget>
  <customwidget>
   <class>StyledLabel</class>
   <extends>QLabel</extends>
   <header>gui/common/StyledLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PostedCreatePostDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
